charts/datadog/templates/NOTES.txt

```diff
@@ -155,24 +155,14 @@ The Datadog Agent is listening on port {{ $apmPort }} for APM service.
 
 {{- end }}
 
-{{- if and .Values.datadog.apm.instrumentation.enabled_namespaces (eq (include "cluster-agent-enabled" .) "false")}}
+{{- if and .Values.datadog.apm.instrumentation.enableddNamespaces (not .Values.datadog.apm.instrumentation.enabled) }}
 
 #################################################################
 ####               WARNING: Configuration notice             ####
 #################################################################
 
-You are using datadog.apm.instrumentation.enabledNamespaces but you disabled the cluster agent. This configuration is unsupported and Kubernetes resource monitoring has been turned off.
-To enable it please set clusterAgent.enabled to 'true'.
-{{- end }}
-
-{{- if and .Values.datadog.apm.instrumentation.enabled .Values.datadog.apm.instrumentation.enabledNamespaces }}
-
-#################################################################
-####               WARNING: Configuration notice             ####
-#################################################################
-
-The options `datadog.apm.instrumentation.enabled` and `datadog.apm.instrumentation.enabledNamespaces` are set together.
-APM Single Step Instrumentation will be enabled in the whole cluster.
+The option `datadog.apm.instrumentation.enabledNamespaces` is set while `datadog.apm.instrumentation.enabled` is disabled.
```

Comment:
Correct me if i'm wrong but the new behaviour is that `datadog.apm.instrumentation.enabled` is true to enable the admission controller and so `datadog.apm.instrumentation.enabledNamespaces` or `datadog.apm.instrumentation.disabledNamespaces` is only took into account if `datadog.apm.instrumentation.enabled: true` 

```suggestion
The option `datadog.apm.instrumentation.enabledNamespaces` can be set while `datadog.apm.instrumentation.enabled` is enabled.
```

Reply:
same change need to be done for `disabledNamespaces` a bit bellow in this file. 

IMO the only WARNING linked to SSI in NOTES.txt can be about the fact that the user can't use at the same time `enabledNamespaces` and `disabledNamespaces` options

Reply:
> IMO the only WARNING linked to SSI in NOTES.txt can be about the fact that the user can't use at the same time enabledNamespaces and disabledNamespaces options

I agree with this in general. But after we changed configuration options, I think it would be useful to notify customers that their existing configuration won't lead to instrumentation. For example, if users used SSI for the past few months to enable SSI in selected namespaces (when apm.instrumentation.enabled=false) and now upgrade to the new chart, showing the warning will help them to understand why they SSI suddenly stopped working.

Reply:
So maybe instead we should add an error/warning when `datadog.apm.instrumentation.enabled == true && len(datadog.apm.instrumentation.enabledNamespaces) == 0 && len(datadog.apm.instrumentation.disabledNamespaces) == 0`

to say that when `datadog.apm.instrumentation.enabled: true` they must configure either `enabledNamespaces` or `disabledNamespaces`. 

WDYT?